<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeftWidget</class>
 <widget class="QWidget" name="LeftWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>134</width>
    <height>379</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="BtnHome">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>125</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>综合信息</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/home256x256.png</normaloff>:/images/home256x256.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnRoadControl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>125</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>道路管控</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/roadControl256x256.png</normaloff>:/images/roadControl256x256.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnDataMonitor">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>125</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>流量监控</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/monitor.jpg</normaloff>:/images/monitor.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCarLocation">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>125</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>车辆定位</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/carLocation.png</normaloff>:/images/carLocation.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCarControl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>125</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>车辆控制</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/carControl.png</normaloff>:/images/carControl.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
